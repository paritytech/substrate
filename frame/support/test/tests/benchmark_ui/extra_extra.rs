use frame_benchmarking::v2::*;
#[allow(unused_imports)]
use frame_support_test::Config;

#[benchmarks]
mod benches {
	use super::*;

	#[benchmark(extra, extra)]
	fn bench() {
		#[block]
		{}
	}
}

fn main() {}
