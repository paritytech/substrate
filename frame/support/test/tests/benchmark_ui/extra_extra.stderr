error: unexpected end of input, `extra` can only be specified once
 --> tests/benchmark_ui/extra_extra.rs:9:26
  |
9 |     #[benchmark(extra, extra)]
  |                             ^
